packages/venia-concept/src/index.css

```diff
@@ -42,6 +42,10 @@ body {
     padding: 0;
 }
 
+.modalIsOpen {
```

Comment:
I couldn't find this used in the code anymore. Can we delete?

Reply:
Removed in latest update